test/string_utils_test.cpp

```diff
@@ -21,6 +21,35 @@
 namespace {
 
 using ::spvutils::CardinalToOrdinal;
+using ::spvutils::ToString;
+
+TEST(ToString, Int) {
+  EXPECT_EQ("0", ToString(0));
+  EXPECT_EQ("1000", ToString(1000));
+  EXPECT_EQ("-1", ToString(-1));
+  EXPECT_EQ("0", ToString(0LL));
+  EXPECT_EQ("1000", ToString(1000LL));
+  EXPECT_EQ("-1", ToString(-1LL));
+}
+
+TEST(ToString, Uint) {
+  EXPECT_EQ("0", ToString(0U));
+  EXPECT_EQ("1000", ToString(1000U));
+  EXPECT_EQ("0", ToString(0ULL));
+  EXPECT_EQ("1000", ToString(1000ULL));
+}
+
+TEST(ToString, Float) {
+  EXPECT_EQ("0", ToString(0.f));
+  EXPECT_EQ("1000", ToString(1000.f));
+  EXPECT_EQ("-1.44", ToString(-1.44f));
```

Comment:
This value has a non-terminating fractional value in binary.  In general it's slightly risky to use such a value in an equality test.   E.g. I would have used something with a binary representation that terminated after fewer than 23 significand bits.

But this is fine, provided the bots pass. :-)